examples/replay/replay_klws_agent.py

```diff
@@ -0,0 +1,120 @@
+import logging
```

Comment:
The specific usefulness of this example is that it could be used to check the determinism of a policy assuming we add the configuration for people to bring their own agent.